assemblies/assembly_designing-and-building-cases/main.adoc

```diff
@@ -13,9 +13,13 @@ include::_artifacts/author-group.adoc[]
 
 As a developer, you can use {CENTRAL} to configure {PRODUCT} assets for case management.
 
-Case management differs from Business Process Management (BPM). It focuses more on the actual data being handled throughout the case rather than on the sequence of steps taken to complete a goal. Case data is the most important piece of information in automated case handling, while business context and decision-making is in the hands of the human case worker.
+Case management differs from Business Process Management (BPM). It focuses more on the actual data being handled throughout the case rather than on the sequence of steps taken to complete a goal. Case data is the most important piece of information in automated case handling, while business context and decision-making are in the hands of the human case worker.
 
-{PRODUCT} includes the *IT_Orders* sample project in {CENTRAL}. This project is referred to throughout this document to explain case management concepts and provide examples.
+//<MH>Dev, in viewing the IT_Orders case project in 7.6, I see that the case definitions are actually referred to as business processes when viewing the sample assets. Should we consider a change in the naming of the assets moving forward to avoid confusion. We are using the term "Case definition" as if it will be listed somewhere.</MH>
```

Comment:
If you mean that you can see it in Business Central, i.e. not the Showcase app, then this is expected as we currently don't have a dedicated UI for case management afaik.

Reply:
I understand :)